notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb

```diff
@@ -2,7 +2,7 @@
   "cells": [
```

Comment:
<p>tabular classification_pipeline.json -&gt; tabular_classification_pipeline.json</p><p> </p><p>replace the space with underscore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>